src/Feature.cpp

```diff
@@ -42,21 +43,43 @@ void Feature::Load(json& o_json)
 	SI_Error rc = ini.LoadFile(ini_path.c_str());
 
 	if (rc < 0) {
-		if (!FeatureIssues::IsObsoleteFeature(GetShortName()))
+		if (FeatureIssues::IsObsoleteFeature(GetShortName())) {
```

Comment:
No. This doesn't make sense. If the file is deleted we don't have to talk about it.